hack/bin/buildah-make-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -ex
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+TOP_LEVEL="$(cd "$DIR/../.." && pwd)"
+
+EXECUTABLES="cannon brig cargohold galley gundeck federator brig-index brig-schema galley-schema galley-migrate-data gundeck-schema proxy spar spar-schema"
+CONTAINER_NAME="output"
+DOCKER_TAG=$USER
```

Comment:
I think we should expect this variable to be set already, as have it in the Makefile. Maybe it can look something like:
```suggestion
DOCKER_TAG=${DOCKER_TAG:-$USER}
```